pkgs/development/libraries/quarto/default.nix

```diff
@@ -0,0 +1,56 @@
+{ callPackage
+, lib
+, pandoc
+, esbuild
+, deno
+, stdenv
+, fetchurl
+, nixos
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "quarto";
+  version = "1.0.38";
+
+  src = fetchurl {
+    url = "https://github.com/quarto-dev/quarto-cli/releases/download/v1.0.38/quarto-1.0.38-linux-amd64.tar.gz";
+    sha256 = "sha256-bqEGRX6ihyrgmIxqkV9jr/2QfWM5LWnsjoGpTqK9qOc=";
+  };
+
+  buildInputs = [
+      pandoc
+      deno
+      esbuild
+  ];
+
+  installPhase = ''
+      mkdir src
+      tar -zxvf ${finalAttrs.src} -C src
+      mkdir -p $out/bin
+      mkdir -p $out/share
+
+      rm src/quarto-${finalAttrs.version}/bin/tools/deno
+      rm src/quarto-${finalAttrs.version}/bin/tools/pandoc
+      rm src/quarto-${finalAttrs.version}/bin/tools/esbuild
+
+      mv src/quarto-${finalAttrs.version}/bin/* $out/bin
+      mv src/quarto-${finalAttrs.version}/share/* $out/share
+
+      ln -s ${deno.outPath}/bin/deno $out/bin/tools/deno
+      ln -s ${pandoc.outPath}/bin/pandoc $out/bin/tools/pandoc
+      ln -s ${esbuild.outPath}/bin/esbuild $out/bin/tools/esbuild
```

Comment:
```suggestion
      ln -s ${deno}/bin/deno $out/bin/tools/deno
      ln -s ${pandoc}/bin/pandoc $out/bin/tools/pandoc
      ln -s ${esbuild}/bin/esbuild $out/bin/tools/esbuild
```

Reply:
Solved in [152051c1949b0717f59d072330ecc83bccd1fe6e](https://github.com/NixOS/nixpkgs/pull/186697/commits/152051c1949b0717f59d072330ecc83bccd1fe6e)

Reply:
It still looks the same as it did before.

Reply:
Do we really need to add these symlinks to `$out/bin/tools` (a really ugly directory to have) instead of wrapping the binaries and adding the dependencies to PATH?

Reply:
Solved in https://github.com/NixOS/nixpkgs/pull/186697/commits/75b5e47c24b42f5bdef512abcdde0709476aa33a

Yes I can create a patch.
Solved in https://github.com/NixOS/nixpkgs/pull/186697/commits/fb7d200e5a9ddad2219e7ff48882ffdcd87114d1